stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/header/MessagesHeaderView.kt

```diff
@@ -195,6 +196,14 @@ public class MessagesHeaderView : ConstraintLayout {
             .build()
     }
 
+    private fun getTextColor(lightColour: Int): Int {
+        return when (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK) {
+            Configuration.UI_MODE_NIGHT_NO -> lightColour
+            Configuration.UI_MODE_NIGHT_YES -> R.color.stream_ui_white
+            else -> lightColour
+        }
```

Comment:
Same question as before, can't we move this to resources and override those colours in the night variant?